src/wallet/wallet_tx_builder.cpp

```diff
@@ -32,38 +32,34 @@ CalcZIP317Fee(
     std::vector<CTxOut> vout{};
     size_t sproutOutputCount{}, saplingOutputCount{}, orchardOutputCount{};
     for (const auto& payment : payments) {
-        std::visit(match {
+        examine(
+            payment.address,
             [&](const CKeyID& addr) {
                 vout.emplace_back(payment.amount, GetScriptForDestination(addr));
             },
             [&](const CScriptID& addr) {
                 vout.emplace_back(payment.amount, GetScriptForDestination(addr));
             },
-            [&](const libzcash::SaplingPaymentAddress&) {
-                ++saplingOutputCount;
-            },
-            [&](const libzcash::OrchardRawAddress&) {
-                ++orchardOutputCount;
-            }
-        }, payment.address);
+            [&](const libzcash::SaplingPaymentAddress&) { ++saplingOutputCount; },
+            [&](const libzcash::OrchardRawAddress&) { ++orchardOutputCount; });
```

Comment:
Note that `AlignAfterOpenBracket: BlockIndent` (as I suggest [here](https://github.com/zcash/zcash/pull/6678/commits/e247b7af79e2c2f88d24b6ba2ea3e90e8f532181#r1200399410)) would make this:
```c++
        examine(
            payment.address,
            [&](const CKeyID& addr) {
                vout.emplace_back(payment.amount, GetScriptForDestination(addr));
            },
            [&](const CScriptID& addr) {
                vout.emplace_back(payment.amount, GetScriptForDestination(addr));
            },
            [&](const libzcash::SaplingPaymentAddress&) { ++saplingOutputCount; },
            [&](const libzcash::OrchardRawAddress&) { ++orchardOutputCount; }
        );
```
which I much prefer.